rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1724,8 +1763,10 @@ raliRandomCrop(
         output_info.height(input->info().height_single());
         output = context->master_graph->create_image(output_info, is_output);
         output->reset_image_roi();
-        context->master_graph->add_node<RandomCropNode>({input}, {output})->init(crop_area_factor, crop_aspect_ratio,
-                                                                           x_drift, y_drift);
+        std::shared_ptr<RandomCropNode> crop_node =  context->master_graph->add_node<RandomCropNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)